src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -66,6 +66,18 @@ public override HttpResponseMessage CreateCheckStatusResponse(HttpRequestMessage
                 functionType: FunctionType.Orchestrator,
                 isReplay: false);
 
+            if (waitUntilOrchestrationStarts)
+            {
+                DurableOrchestrationStatus status = await this.GetStatusAsync(instance.InstanceId);
+                int counter = 0;
+                while ((status == null || status.RuntimeStatus == OrchestrationRuntimeStatus.Pending) && counter < 20)
+                {
+                    await Task.Delay(500);
```

Comment:
I would reduce this a little bit - maybe 200 milliseconds?

Reply:
Sounds great! Initially I configured 300 ms but 200 ms sounds even better.